docs/sdks/atc.md

```diff
@@ -0,0 +1,333 @@
+    const tws = {txn: ptxn, signer: signer}
+
+    // Pass TransactionWithSigner to ATC
+    atc.addTransaction(tws)
+
+    ```
+
+=== "Go"
+    ```go
+
+    // ...
+
+    // Returns Account 
+    acct, _ := GetAccount()
+
+    // Create signer object
+	signer := future.BasicAccountTransactionSigner{Account: acct}
+
+    // Get suggested params from client
+	sp, _ := client.SuggestedParams().Do(context.Background())
+
+    // Create a transaction
+	ptxn, _ := future.MakePaymentTxn(acct.Address.String(), acct.Address.String(), 10000, nil, "", sp)
+
+    // Construct TransactionWithSigner
+	tws := future.TransactionWithSigner{Txn: txn, Signer: signer}
+
+    // Pass TransactionWithSigner to atc
+
+    atc.AddTransaction(tws)
+
+    ```
+
+=== "Java"
+    ```java
+
+    // ...
+
+    // Returns Account 
+    Account acct = getAccount();
+
+    // Create signer object
+	BasicAccountTransactionSigner signer = new BasicAccountTransactionSigner(acct);
+
+    // Get suggested params from client
+    Response<TransactionParametersResponse> rsp = client.TransactionParams().execute();
+    TransactionParametersResponse sp = rsp.body();
+
+    // Create a transaction
+    Transaction ptxn = PaymentTransactionBuilder.Builder().amount(10000).suggestedParams(sp)
+            .sender(acct.getAddress()).receiver(acct.getAddress()).build();
+
+    // Construct TransactionWithSigner
+	TransactionWithSigner tws = new TransactionWithSigner(ptxn, signer);
+
+    // Pass TransactionWithSigner to atc
+    atc.addTransaction(tws);
+
+    ```
+
+The call to add a transaction may may be performed multiple times, each time adding a new transaction to the atomic group. Recall that a maximum of 16 transactions may be included in a single group.
+
+
+## Calling ABI Methods
+
+When calling an [ABI](../get-details/dapps/smart-contracts/ABI/index.md) compliant application, the Atomic Transaction Composer will handle encoding and decoding of the arguments passed and the return value.  It will also make sure that any [reference types](../get-details/dapps/smart-contracts/ABI/index.md#reference-types) are packed into the transaction group appropriately.  Additionally, since it knows the method signature and types required, it will do some type checking to make sure the arguments passed are valid for the method call. 
+
+In order to call the methods, a Contract or Interface is constructed. Typically this will be done using a [json file](../get-details/dapps/smart-contracts/ABI/index.md#api) that describes the api for the application.   
+
+Once the Contract object is constructed, it can be used to look up and pass method objects into the Atomic Transaction Composers `add_method_call`
+
+=== "Python"
+    ```py
+    from algosdk.abi import Contract
+
+    with open("path/to/contract.json") as f:
+        js = f.read()
+    c = Contract.from_json(js)
+
+    # Using the app id from the "sandnet" network, which is hardcoded in the json file
+    app_id = c.networks["sandnet"].app_id
```

Comment:
I was under the impression the networks would be the genesis hash rather than the name?